pyproject.toml

```diff
@@ -0,0 +1,58 @@
+[build-system]
+requires = ["hatchling",
+"hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "execnet"
+dynamic = ["version"]
+description = "execnet: rapid multi-Python deployment"
+long_description_file = "README.rst"
+license = "MIT"
+requires-python = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*"
+authors = [
+    { name = "holger krekel and others" },
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: POSIX",
+    "Programming Language :: Python :: 2",
+    "Programming Language :: Python :: 2.7",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.5",
```

Comment:
3.5 is failing on CI, I'm not opposed to dropping support for <3.7 in the next release.

Reply:
i'm about to drop all the old ones